app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you implement the callback suggestion above here as well.